Kinvey/Kinvey/Geolocation.swift

```diff
@@ -14,6 +14,8 @@ import MapKit
 /// Class that represents a 2D geolocation with latitude and longitude
 public final class GeoPoint: Object {
     
+    public static let CodingKey = "_geoloc"
```

Comment:
What is `_geoloc` used for in the multi-insert implementation?

Reply:
some tests are related to send invalid geolocations, like (-300, -300)
since this is a special field handled in the backend I decided to create this constant so we (and also developers) don't have to repeat `_geoloc` everywhere